src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -120,6 +130,26 @@ def test_conversion_as_lone_interface(self):
                 self.run_custom_checks()
 
 
+def _create_mock_probe_for_recording(recording):
+    import probeinterface as pi
+    import spikeinterface as si
+
+    R: si.BaseRecording = recording
+    n = R.get_num_channels()
+    positions = np.zeros((n, 2))
+    for i in range(n):
+        x = i // 4
+        y = i % 4
+        positions[i] = x, y
+    positions *= 20
+    probe = pi.Probe(ndim=2, si_units="um")
+    probe.set_contacts(positions=positions, shapes="circle", shape_params={"radius": 5})
+    probe.set_device_channel_indices(np.arange(n))
+    # set shank ids as 0, 1, 2, 0, 1, 2, ...
+    probe.set_shank_ids(np.arange(n) % 3)
```

Comment:
a bit odd to have electrodes that are near each other be assigned to different shanks, but I don't think this really poses a problem for now. We can always improve this to be more realistic later if needed

Reply:
With my latest commit I have adjusted this to be more realistic.